modules/dnn/test/test_tf_importer.cpp

```diff
@@ -1058,8 +1058,20 @@ TEST_P(Test_TensorFlow_layers, keras_mobilenet_head)
 
 TEST_P(Test_TensorFlow_layers, resize_bilinear)
 {
-    runTensorFlowNet("resize_bilinear");
-    runTensorFlowNet("resize_bilinear_factor");
+    runTensorFlowNet("resize_bilinear");        // TF case: align_corners=False, half_pixel_centers=False
+    runTensorFlowNet("resize_bilinear",
+                     false, 0.0, 0.0, false,    // default parameters
+                     "_align_corners");         // TF case: align_corners=True, half_pixel_centers=False
```

Comment:
Please do 
```cpp
runTensorFlowNet("resize_bilinear_align_corners")
```
instead of introducing `groupPrefix`.

Also, split every new test case into separate scope:
```
TEST_P(Test_TensorFlow_layers, resize_bilinear_align_corners)
{
    runTensorFlowNet("resize_bilinear_align_corners");
}
```

Reply:
What about input data duplications? `groupPrefix` was introduced in order to reuse the same input data for test groups, where the group is, for example, `resize_bilinear` and derived cases with different parameters. Thus, we can reduce extra input data generation in opencv-extra (also added appropriate changes in generation script). In perspective, it can be useful.

Reply:
Agree, thanks for clarification! So please just move them into different scopes (this way it's simpler to track failed tests)

Reply:
Tests were split.